src/app/cli/src/coda_main.ml

```diff
@@ -1092,8 +1092,7 @@ module Run (Config_in : Config_intf) (Program : Main_intf) = struct
     ; peers= List.map (peers t) ~f:(fun (p, _) -> Host_and_port.to_string p)
     ; user_commands_sent= !txn_count
     ; run_snark_worker= run_snark_worker t
-    ; proposal_interval=
-        Option.value (Unix.getenv "CODA_PROPOSAL_INTERVAL") ~default:"5000"
+    ; proposal_interval= Int64.to_int_exn Consensus.Mechanism.block_interval_ms
```

Comment:
Really, you don't even need to store this in this record, you could just reference `Consensus.Mechanism.block_interval_ms` in `daemon_rpcs.ml`.

Reply:
it also needs to be in the record to come out in the JSON output

Reply:
for `status --json`